public/coffee/ide/human-readable-logs/HumanReadableLogsRules.coffee

```diff
@@ -88,4 +88,52 @@ define -> [
 			You have used a font command which is only available in math mode. To use this command, you must be in maths mode (E.g. $ \u2026 $ or \\begin{math} \u2026 \\end{math}). If you want to use it outside of math mode, use the text version instead: \\textrm, \\textit, etc.
 		"""
+	,
+		regexToMatch: /LaTeX Error: Can be used only in preamble/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/LaTeX_Error:_Can_be_used_only_in_preamble"
+		humanReadableHint: """
+			You have used a command in the main body of your document which should be used in the preamble. Make sure that \\documentclass[\u2026]{\u2026} and all \\usepackage{\u2026} commands are written before \\begin{document}.
+		"""
+	,
+		regexToMatch: /Missing \\right inserted/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/Missing_%5Cright_insertede"
+		humanReadableHint: """
+			You have started an expression with a \\left command, but have not included a corresponding \\right command. Make sure that your \\left and \\right commands balance everywhere, or else try using \\Biggl and \\Biggr commands instead as shown <a target=\"_blank\" href=\"https://www.sharelatex.com/learn/Errors/Missing_%5Cright_inserted\">here</a>.
+		"""
+	,
+		regexToMatch: /Double superscript/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/Double_superscript"
+		humanReadableHint: """
+			You have written a double superscript incorrectly as a^b^c, or else you have written a prime with a superscript. Remember to include { and } when using multiple superscripts.
+		"""
+	,
+		regexToMatch: /Double subscript/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/Double_subscript"
+		humanReadableHint: """
+			You have written a double subscript incorrectly as a_b_c. Remember to include { and } when using multiple subscripts.
+		"""
+	,
+		regexToMatch: /No \\author given/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/No_%5Cauthor_given"
+		humanReadableHint: """
+			You have used the \\maketitle command, but have not specified any \\author. To fix this, include an author in your preamble using the \\author{\u2026} command.
+		"""
+	,
+		regexToMatch: /LaTeX Error: Environment .+ undefined/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors%2FLaTeX%20Error%3A%20Environment%20XXX%20undefined"
+		humanReadableHint: """
+			You have created an environment (using \\begin{\u2026} and \\end{\u2026} commands) which is not recognized. Make sure you have included the required package for that environment in your preamble, and that the environment is spelled correctly.
+		"""
+	,
+		regexToMatch: /LaTeX Error: Something's wrong--perhaps a missing \\item/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/LaTeX_Error:_Something%27s_wrong--perhaps_a_missing_%5Citem"
+		humanReadableHint: """
+			There are no entries found in a list you have created. Make sure you label list entries using the \\item command, and that you have not used a list inside a table incorrectly.
```

Comment:
Can lists be used in tables at all? If there's a correct way to do it, is it possible to say what here?